### PR TITLE
Community docs

### PR DESCRIPTION
**Pull request by Gimpneek**
Adding files to help people install NHClinical, contribute code and feedback and standards of communications

---

Before this pull request can be merged the following must be true.
- [ ] Unit tests pass (Travis integration).
- [ ] No code quality issues (Codacy integration).
- [ ] Approval from at least one developer and at least one tester.

If you are a BJSS contributor there are some additional conditions.
- [ ] All client module unit tests pass.
- [ ] The *Pull Request* field in the work tab of the JIRA issue is populated.
- [ ] The JIRA issue contains a description of the root cause and the solution in the comments.
- [ ] There are no bugs against the JIRA issues in the current sprint.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,45 @@
...
+
+After restarting the server you can then log in as the admin user and install the
+`nh_clinical` module. This installs the different models you can work with.
+
+## Upgrading
+To upgrade the NHClinical modules you just need to update the NHClinical modules and
+press the upgrade module button for `nh_clinical`.
+
+It's recommended you backup the existing database and module files before upgrading
+so you can restore should anything go wrong.
```
Comment (moderatemisbehaviour) [inline README.md]: Reading this from an uninformed point of view I'd probably be thinking what's the difference between _upgrading_ and _updating_ a module and where / what is the _upgrade module button_?

I think people are either going to be familiar enough with Odoo to know how upgrading works or they won't know anything at all. These instructions assume a bit of knowledge and so are a middle-ground.

I'd say we need to explain more in-depth about what addons are, that they go in directories specified in the server config, then you have to log in as Admin, and navigate to the modules page. This long-form content would probably be more appropriate in a wiki and will also be appropriate for the **openeobs** repo.

Personally I would write it up as a wiki page in the **openeobs** repo and replace this *Upgrading* section with a recommendation that people check out the **openeobs** wiki for more information about topics such as upgrading. I don't think there's really much to say in the wiki for **nhclinical**.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+We welcome any feedback, suggestions and bugs that you have, however we ask 
+that when submitting bugs please fill out the following:
```
[the remark this answers is not in the document]
Comment (Gimpneek) [reply .github/ISSUE_TEMPLATE.md]: Added

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+## Description
+*What functionality did the bug impact, how did you get to the area
+of the app that the bug was found in and what were you trying to do at the time*
+
+## Steps to reproduce
+*A list of steps navigating through the app to reproduce the bug*
```
[the remark this answers is not in the document]
Comment (Gimpneek) [reply .github/ISSUE_TEMPLATE.md]: Need to drive home that they should tell us how they find it

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+## Actual Result
+*What happened when you triggered the bug?*
+
+## Expected Result
+*What were you expecting to happened?*
```
[the remark this answers is not in the document]
Comment (Gimpneek) [reply .github/ISSUE_TEMPLATE.md]: Changed

File: README.md [hunk 5]
```diff
@@ -1,2 +1,45 @@
...
+- Patient Visits
+- Ward Staff Management
+
+These models create a basic hospital set up but it's recommended to install
+[Open-eObs](https://github.com/NeovaHealth/openeobs) which adds all the functionality
+needed to run an Acute or Mental Health hospital.
+
+## Installation
+NHClinical is a set of [Odoo](https://www.odoo.com/) modules that add clinical models 
+and functionality to Odoo.
```
[the remark this answers is not in the document]
Comment (Gimpneek) [reply README.md]: Moved to top of file

File: README.md (hunk 1 above)
Comment (Gimpneek) [reply README.md]: Sure will replace with Wiki page

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (moderatemisbehaviour) [reply .github/ISSUE_TEMPLATE.md]: _Repeat for emphasis_. Fair does.

**Pull request closed without merging**